src/backends/postgresql/statement.cpp

```diff
@@ -768,15 +768,18 @@ void postgresql_statement_backend::describe_column(int colNum, data_type & type,
         type = dt_double;
         break;
 
-    case 16:   // bool
     case 21:   // int2
+        type = dt_int16;
+        break;
+
+    case 16:   // bool
```

Comment:
Is `bool` really 4 bytes in PostgreSQL? [The docs](https://www.postgresql.org/docs/current/datatype-boolean.html) seem to be saying that it's 1, but I could be mixing up something here.

Reply:
You're right, I didn't check that.